README.md

```diff
@@ -108,6 +108,17 @@ such a server can be launched with `python -m smtpd -n -c DebuggingServer localh
 
 Alternatively only unit tests can be run by doing `cargo test --lib`.
 
+## Troubleshooting
+
+These are general steps to be followed when troubleshooting SMTP related issues.
+- Ensure basic connectivity, ensure requisite ports are open and daemons are listening.
+- Confirm that your service provider allows traffic on the ports being used for mail transfer. (traffic on ports 25, 2525, 587, and 465 is commonly blocked by some service providers.)
+- Check if your service provider requires using an SMTP relay, and ensure you are correctly authenticated to use this relay.
```

Comment:
:thinking: lettre always requires using an SMTP relay

Reply:
Yeah I couldn't figure how or even if I should have included this part, but generally I would want the docs to persuade users to troubleshoot all other possibilities before assuming issues with lettre exist.

Reply:
I'm going to remove the "check" and just leave the "ensure you are authenticating correctly".

Reply:
I've removed the "check" verbiage, and simplified it to `- Check SMTP relay authentication and configuration.`

Reply:
I'm not certain of the etiquette for reviews, I will resolve this conversation now that I've made this change.